channels/rdpgfx/server/rdpgfx_main.c

```diff
@@ -330,6 +340,11 @@ static UINT rdpgfx_send_create_surface_pdu(RdpgfxServerContext* context,
 {
 	wStream* s = rdpgfx_server_single_packet_new(RDPGFX_CMDID_CREATESURFACE, 7);
 
+	WINPR_ASSERT(context);
+	WINPR_ASSERT(pdu);
+	WINPR_ASSERT((pdu->pixelFormat == GFX_PIXEL_FORMAT_XRGB_8888 0x20) ||
```

Comment:
strange statement `(pdu->pixelFormat == GFX_PIXEL_FORMAT_XRGB_8888 0x20)` ?

Reply:
true that.